elasticsearch/templates/statefulset.yaml

```diff
@@ -276,3 +283,7 @@ spec:
         {{- end }}
       {{- end }}
       {{- end }}
+{{- if .Values.lifecycle }}
```

Comment:
Could you add a test for this please. It will look something like:

```
def test_lifecycle_hooks():
    config = ''
    r = helm_template(config)
    c = r['statefulset'][uname]['spec']['template']['spec']['containers'][0]
    assert 'lifecycle' not in c

    config = '''
    lifecycle:
      preStop:
        exec:
          command: ["/bin/bash","/preStop"]
    '''
    r = helm_template(config)
    c = r['statefulset'][uname]['spec']['template']['spec']['containers'][0]

    assert c['lifecycle']['preStop']['exec']['command'] == ["/bin/bash","/preStop"]
```